docs/working-with-projects/working-with-images/create-tag-immutability-rules.md

```diff
@@ -3,46 +3,46 @@ title: Tag Immutability Rules
+Since v2.0, you can delete any tag of an artifact without deleting the artifact itself or any of its other tags. Therefore, you can lock down a particular tag by configuring an immutability rule matching this tag which means the artifact holding the tag also cannot be overwritten or deleted. However you can still delete other tags associated with this immutable artifact. Consider the follow example:
 
-1. In a project, set an immutable tag rule that matches the image and tag `hello-world:v1`.
-1. In the Docker client, pull `hello-world:v1` and retag it to `hello-world:v2`.
-1. Push `hello-world:v2` to the same project.
-1. In the Harbor interface, attempt to delete `hello-world:v2`.
+1. In the Docker client, push `hello-world:v1` into a project.
+1. In the project, set an immutable tag rule in this project that matches the image and tag `hello-world:v1`.
+1. Push `hello-world:v1` to the project.
+1. In your local env, retag `hello-world:v1` to `hello-world:v2`.
+1. Push `hello-world:v2` to the project.
+1. In the Harbor interface, attempt to delete tag `v1` and `v2` of `hello-world` sequentially.
 
-In this case, you cannot delete `hello-world:v2` because it shares the sha256 digest with `hello-world:v1`, and `hello-world:v1` is an immutable tag. 
+In this case, you cannot delete tag `v1` as it's an immutable tag and you cannot delete the artifact `hello-world` holding this tag. But you can delete tag `v2` even it shares the sha256 digest with `v1`. 
 
 ## Create a Tag Immutability Rule
 
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
-1. Go to **Projects**, select a project, and select **Tag Immutability**.
+1. Go to **Projects**, select a project, select policy, and select **Tag Immutability**.
 
-    ![Add an immutability rule](../../../img/tag-immutability.png)
+    ![Add an immutability rule](../../img/tag-immutability.png)
 
 1. Click **Add Rule**.
 
     - In the **Respositories** row, enter a comma-separated list of repositories to which to either apply or exclude from the rule by selecting either **matching** or **excluding** from the drop-down menu.
     - In the **Tags** row, enter a comma-separated list of tags to which to either apply or exclude from the rule by selecting either **matching** or **excluding** from the drop-down menu.
  
-      ![Add an immutability rule](../../../img/add-immutability-rule.png)
+      ![Add an immutability rule](../../img/add-immutability-rule.png)
 1. Click **Add** to save the rule.
 
     You can add a maximum of 15 immutability rules per project. 
 
     After you add a rule, any tags that are identified by the rule are marked **Immutable** in the Repositories tab.
 1. To modify an existing rule, use the **Action** drop-down menu next to a rule to disable, edit, or delete that rule. 
 
-    ![Immutability rules](../../../img/edit-tag-immutability.png)
+    ![Immutability rules](../../img/edit-tag-immutability.png)
```

Comment:
‘../../../' might be correct, it's always one level more than correct directory structure, please make sure

Reply:
updated